default.nix

```
@@ -8,11 +8,12 @@
 , network ? "mainnet"  # mainnet|testnet
 , rev ? "dirty"
 , static ? stdenv.hostPlatform.isStatic
+, nativeByteOrder ? true # nativeByteOrder mode will panic on big endian machines
 }:
 let
   version = "v1.0.4";
   pname = "cronosd";
-  tags = [ "ledger" "netgo" network "rocksdb" "grocksdb_no_link" ];
+  tags = [ "ledger" "netgo" network "rocksdb" "grocksdb_no_link" ] + lib.optional nativeByteOrder "nativebyteorder";
```

seems types is not right
```suggestion
  tags = [ "ledger" "netgo" network "rocksdb" "grocksdb_no_link" ] ++ (if nativeByteOrder then [ "nativebyteorder" ] else []);
```

---

fixed